phone/src/Phone.tsx

```diff
@@ -72,7 +70,7 @@ function Phone() {
               <Route exact path="/" component={HomeApp} />
               {callModal && <Route exact path="/call" component={CallModal} />}
               {apps.map((App) => (
-                <>{!App.isDisabled && <App.Route key={App.id} />}</>
+                <Fragment key={App.id}>{!App.isDisabled && <App.Route key={App.id} />}</Fragment>
```

Comment:
Fixing key error in "Phone.tsx"

Reply:
You may want to add this fix also in `ui/components/GridMenu.tsx`.

Reply:
Sure